test/Makefile

```diff
@@ -82,6 +82,11 @@ $(INITRAMFS)/bin:
 	@mkdir -p $@
 	@/bin/busybox --install -s $@
 
+$(INITRAMFS)/sbin:
+	@mkdir -p $@
+	@cp /sbin/ldconfig $@
+	@cp /sbin/ldconfig.real $@
```

Comment:
What is `ldconfig.real`? I googled it, I but didn't get not many useful results.

Reply:
`ldconfig` is a shell script used to scan system directories and create cache files. During this process, the script invokes the binary file `ldconfig.real` to complete the creation of the cache files.